### PR TITLE
feat: Implement admin -> messages endpoint

### LINKED ISSUE #1504: Implement Admin -> Messages endpoint
**Is your feature request related to a problem? Please describe.**
The admin -> messages table has dummy data right now. After https://github.com/fossasia/open-event-server/pull/5213 is merged, the endpoint can be implemented here.

**Additional context**
To be done after https://github.com/fossasia/open-event-server/pull/5213 is merged

### PR DESCRIPTION
**Pull request by mayank8318**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Implements admin -> messages endpoint.

#### Changes proposed in this pull request:

- Add model for `message-setting`
- Add the required cells for displaying the table
screenshot from 2018-07-28 13-35-26

Save action to be implemented in the next PR.

Fixes #1504

### CODE REVIEW

File: app/components/ui-table/cell/admin/messages/cell-notification-message.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+});
```
Comment (ritikamotwani) [inline app/components/ui-table/cell/admin/messages/cell-notification-message.js]: @mayank8318 these files can be removed.

File: app/controllers/admin/messages.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      propertyName : 'recipient',
+      title        : 'Recipients'
+    },
+    {
+      propertyName : 'action',
+      title        : 'Trigger'
+    },
+    {
+      propertyName : '',
```
Comment (ritikamotwani) [inline app/controllers/admin/messages.js]: @mayank8318 why is the property name blank? 'emailSubject' should be the property name and can directly be used in the hbs file.

Comment (mayank8318) [reply app/controllers/admin/messages.js]: But can I pass more than one properties?

Comment (mayank8318) [reply app/controllers/admin/messages.js]: And what you said doesn't work. I will have to do declare something like this ->
```javascript
    {
      emailSubject : this.get('record.emailSubject'),
     emailMessage: this.get('record.emailMessage'),
      title        : 'Email Message',
      template     : 'components/ui-table/cell/admin/messages/cell-email-message'
    }
```

Comment (mayank8318) [reply app/controllers/admin/messages.js]: Sorry event this doesn't work. Can you give any example where this has been done?

Comment (ritikamotwani) [reply app/controllers/admin/messages.js]: No not like this.
```javascript
 {
      propertyName : 'startsAt',
      template     : 'components/ui-table/cell/cell-simple-date',
      dateFormat   : 'MMMM DD, YYYY - HH:mm A',
      title        : 'Date'
    }
```
and then in the template 
```hbs
{{#if (get record column.propertyName)}}
  <span>
    {{moment-format (get record column.propertyName) (if column.dateFormat column.dateFormat 'MMMM DD, YYYY - HH:mm A')}}
  </span>
{{else}}
  <span>
    {{t 'No dates available.'}}
  </span>
{{/if}}
```

**Pull request merged**